Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -8,8 +8,11 @@ namespace Microsoft.Dafny;
 
 public class CommonOptionBag {
 
-  public static readonly Option<int> ErrorLimit =
-    new("--error-limit", () => 5, "Set the maximum number of errors to report (0 for unlimited).");
+  public static readonly Option<int> VerificationErrorLimit =
+    new("--error-limit", () => 5, "Set the maximum number of verification errors to report (0 for unlimited).");
+
+  public static readonly Option<Boogie.CoreOptions.VerbosityLevel> Verbosity =
+    new("--verbosity", () => CoreOptions.VerbosityLevel.Normal, "Set the level of output produced. Ranges from nothing (Silent) to detailed (Trace). With 'Normal', produce warnings, errors, and some extra information. With 'Quiet', produce only errors and warnings.");
```

Comment:
The definition of VerbosityLevel is somehow not in these diffs.
Between Normal and Trace there should be a Verbose level which matches whatever the previous --verbose option did. To me Trace connotes even more output - detailed events along the way.

Reply:
The previous `--verbose` option affected only compilation, and was more like verification tracing, so it seemed like it made sense to put it in the same bucket.

Reply:
I still would like a level in between. The step from Normal to Trace seems huge. I don't expect anyone to use TRACE as a matter of course -- just when needing to understand what is going wrong  with something. Whereas verbose is just a bit more information than normal, but not an overwhelming amount.

For example, I need a way to turn on explanations of error messages on the command-line (like we are putting into hover text in the IDE). Not on by default. But then if accompanied by lots of other 'tracing' information would be too much.